dkg/pedersen/mod_test.go

```diff
@@ -52,15 +52,17 @@ func TestPedersen_Setup(t *testing.T) {
 	_, err = actor.Setup(fakeAuthority, 1)
```

Comment:
😇  how do you feel about splitting tests?

Reply:
for DKG, that will be on @nkcr hands, he's the owner of the package !